cypress/userNumber.js

```diff
@@ -1,11 +1,7 @@
-// This makes a user number which will be used for all tests
-// it needs to happen before startup rather than in support/index.js so
-// that we have the same user number for all feature files
-// This code is therefore called from start.sh.  
-// For safety, for now we generate userNumber using 
-// the exact same js code that the Casper tests did
-var random = Math.floor(Math.random() * 999999999);
-
-var date = new Date();
-var userNumber = date.getTime() + "" + random;
-console.log(userNumber);
+// This makes a user number for tests.
+// It needs to happen before startup rather than in support/index.js so
+// that we have the same user number for all feature files under CI.
+// This code is therefore called from start.sh.
+// For safety, for now we generate userNumber using
+// the exact same js code that the Casper tests did.
+console.log("" + (new Date()).getTime() + Math.floor(Math.random() * 999999999));
```

Comment:
Do ya really think that's more readable than what was there before?? ;)

Reply:
No, I think I'll revert it